storage/tests/tests/test_compact.py

```diff
@@ -1,17 +1,19 @@
 import pytest
 
 from python.src import consts
+from python.src.norcow import NC_CLASSES
 
 from . import common
 
 
-def test_compact():
-    sc, sp = common.init(unlock=True)
+@pytest.mark.parametrize("nc_class", NC_CLASSES)
+def test_compact(nc_class):
+    sc, sp = common.init(nc_class, unlock=True)
     for s in (sc, sp):
         s.set(0xBEEF, b"hello")
         s.set(0xBEEF, b"asdasdasdasd")
         s.set(0xBEEF, b"fsdasdasdasdasdsadasdsadasdasd")
-        s.set(0x0101, b"a" * (consts.NORCOW_SECTOR_SIZE - 600))
+        s.set(0x0101, b"a" * (consts.NORCOW_SECTOR_SIZE - 1200))
```

Comment:
I assume the constant was changed because the test was failing with the old one. But why the test was failing with the old constant?

Reply:
on the blockwise (with 4word blocks) flash the items take more space so in order for all the requested item to fit we need larger reserve.

this however caused the bitwise test not to test compacting properly, so i added more parameters, and also asserts to ensure that the compacting is indeed tested: 

https://github.com/trezor/trezor-firmware/commit/ae40dd3108e31de93be3217404cdffebe7aa0534